From 6bae26909e4d9ce0e58f5562a2c3440cc74c45ae Mon Sep 17 00:00:00 2001
Message-ID: <6bae26909e4d9ce0e58f5562a2c3440cc74c45ae.1747779548.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1747779547.git.sharlatanus@gmail.com>
References: <cover.1747779547.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 20 May 2025 12:30:12 +0100
Subject: [PATCH 28/31] gnu: python-sunkit-magex: Update to 1.1.0.

* gnu/packages/astronomy.scm (python-sunkit-magex): Update to 1.1.0.
[arguments] <test-flags>: Run all tests.
[native-inputs]: Remove python-setuptools-scm; add
python-setuptools-scm-next.

Change-Id: I88533726a2164888f6e6255d4ac8cff181e30036
---
 gnu/packages/astronomy.scm | 16 ++++------------
 1 file changed, 4 insertions(+), 12 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 9777df7cea..26d357b030 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -7637,26 +7637,18 @@ (define-public python-sunkit-image
 (define-public python-sunkit-magex
   (package
     (name "python-sunkit-magex")
-    (version "1.0.0rc1")
+    (version "1.1.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "sunkit_magex" version))
        (sha256
-        (base32 "10lldb2q71q9gip9sx35zyzm34p9aj4xr2c7a21x176q90r3qyqq"))))
+        (base32 "16wraf2yrqslik4h3mhj5gcwx7fdg3x9v38swlhcgcyvkcjxbqc2"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
-      #~(list "--numprocesses" (number->string (min 8 (parallel-job-count)))
-              "-k" (string-join
-                    ;; Some issue with presision in this test.
-                    (list "not test_adapt_map"
-                          ;; NotImplementedError: is_full_sun_synoptic_map is
-                          ;; only implemented for ['CEA', 'CAR'] projections
-                          ;; and not
-                          "test_car_reproject")
-                    " and not "))
+      #~(list "--numprocesses" (number->string (min 8 (parallel-job-count))))
       #:phases
       #~(modify-phases %standard-phases
           (add-before 'check 'pre-check
@@ -7671,7 +7663,7 @@ (define-public python-sunkit-magex
            python-pytest-xdist
            python-reproject
            python-setuptools
-           python-setuptools-scm
+           python-setuptools-scm-next
            python-streamtracer
            python-sympy
            python-wheel))
-- 
2.49.0

